zulipterminal/ui_tools/views.py

```diff
@@ -1076,6 +1076,9 @@ def __init__(
         super().__init__(controller, widgets, "EXIT_POPUP", width, title)
 
 
+# views.py
+
+
```

Comment:
Is there a specific reason for these additions?

Other than that, LGTM

Reply:
ahh no. sorry !
they are there bye mistake

Reply:
@rsashank  can u please guide me on how to remove them and still keep the commits to only one.

Reply:
You can make the change and squash it into your first commit. Alternatively, you could rebase and edit the commit.

Look into [interactive rebase](https://docs.github.com/en/get-started/using-git/using-git-rebase-on-the-command-line).